libbeat/docs/overview.asciidoc

```diff
@@ -12,7 +12,9 @@ for capturing:
 [horizontal]
 Audit data:: https://www.elastic.co/products/beats/auditbeat[Auditbeat]
 Log files:: https://www.elastic.co/products/beats/filebeat[Filebeat]
+Cloud data:: https://www.elastic.co/products/beats/functionbeat[Functionbeat]
 Availability:: https://www.elastic.co/products/beats/heartbeat[Heartbeat]
+Systemd journals:: https://www.elastic.co/downloads/beats/journalbeat[Journalbeat]
```

Comment:
there is no product page for Journalbeat so I've used the download URL.  In the other cases, I think it's nice for users to go to the product pages before downloading.

Reply:
@acchen97 we will need to create one for Journalbeat.